tests/models/test_taskinstance.py

```diff
@@ -2268,6 +2271,11 @@ def test_outlet_assets(self, create_task_instance):
         dagbag = DagBag(dag_folder=example_assets.__file__)
         dagbag.collect_dags(only_if_updated=False, safe_mode=False)
         dagbag.sync_to_db(session=session)
+
+        asset_models = session.scalars(select(AssetModel)).all()
```

Comment:
Likely not a good idea. This table can be quite big since it contains all history. (Same for the query below.)